src/EventExport/Command/ExportEventsAsPDFJSONDeserializer.php

```diff
@@ -82,6 +83,13 @@ public function deserialize(string $data): ExportEventsAsPDF
             $command = $command->withSelection($json->selection);
         }
 
+
+        $sortOrder = SortOrder::fromJson($json);
```

Comment:
Maybe `fromJson` is a bit misleading. `fromJsonData` is more correct probably.

Reply:
No preference, I think `fromJson` is also very clear